src/main/java/com/greenfiling/smclient/model/SupplierCost.java

```diff
@@ -0,0 +1,86 @@
+package com.greenfiling.smclient.model;
+
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+
+public class SupplierCost {
+
+  private Integer id;
+  private String zipcode;
+  @SerializedName(value = "job_type_id")
+  private String jobTypeId;
+  @SerializedName(value = "sla_id")
+  private String slaId;
+  private Double amount;
+  @SerializedName(value = "suggested_retail_price")
+  private Double suggestedRetailPrice;
+  @SerializedName(value = "created_at")
+  private Date createdAt;
```

Comment:
One other comment I didn't notice earlier - I think the standard for dates in this library is java.time.OffsetDateTime.  If you just picked Date out of inertia please change.  If you think there's a compelling reason to use Date instead of OffsetDateTime, let's discuss

Reply:
I was looking through the model objects for a different issue and was reminded of the difference.  We use Date to represent pure dates (eg, from Invoice, `"issued_on": "2015-05-12"`) and OffsetDateTime for date+time+offset (eg, from LineItem, `"updated_at": "2015-03-18T10:19:27-04:00"`).

Reply:
total inertia, thanks for catching that